### PR TITLE
Follow Button

### PR DESCRIPTION
**Pull request by tissyane**
- Created Follow Button as an exported component;
- Inserted follow button at userPosts Page but not at the loggedUserPosts Page;

### CODE REVIEW

File: src/pages/UserPosts.js [hunk 1]
```diff
@@ -8,17 +8,22 @@ import { Title } from "../styles/commons/Title.js";
...
-  const { config, refresh} = useContext(LoginContext);
+  const { config, refresh, userData} = useContext(LoginContext);
   const navigate = useNavigate();
+ 
 
   useEffect(() => {
     getUserPosts(id, config)
       .then((res) => {
+        console.log(id)
+        console.log(userData.userId)
```
Comment (guilhermeicarofr) [inline src/pages/UserPosts.js]: We should remove this console.log()

File: src/pages/UserPosts.js [hunk 2]
```diff
@@ -8,17 +8,22 @@ import { Title } from "../styles/commons/Title.js";
...
   const { id } = useParams();
-  const { config, refresh} = useContext(LoginContext);
+  const { config, refresh, userData} = useContext(LoginContext);
   const navigate = useNavigate();
+ 
 
   useEffect(() => {
     getUserPosts(id, config)
       .then((res) => {
+        console.log(id)
```
Comment (MacenaAndre) [inline src/pages/UserPosts.js:25]: Remove console.logs

**Review by guilhermeicarofr — APPROVED** [review]
Approved, I only noticed a console.log() to be remove

File: src/pages/UserPosts.js (hunk 1 above)
Comment (MacenaAndre) [inline src/pages/UserPosts.js]: remove console.logs

**Review by Francksuel — APPROVED** [review]

**Review by MacenaAndre — APPROVED** [review]
Great!

**Pull request merged**